package.json

```diff
@@ -62,35 +62,23 @@
     "jupyterlab-tour": "^3.1.4"
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
     "@jupyterlab/builder": "^3.1.0",
-    "@jupyterlab/testing": "^4.0.0-alpha.17",
-    "@jupyterlab/testutils": "^4.0.0-alpha.1",
-    "@types/jest": "^26.0.0",
-    "@types/node": "^18.11.17",
-    "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
     "@typescript-eslint/eslint-plugin": "^4.8.1",
     "@typescript-eslint/parser": "^4.8.1",
     "eslint": "^7.14.0",
     "eslint-config-prettier": "^6.15.0",
     "eslint-plugin-prettier": "^3.1.4",
-    "jest": "^26.0.0",
     "node-fetch": "^3.3.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.1.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-joyride": "^2.5.3",
+    "react-joyride": "2.3.0",
     "rimraf": "^3.0.2",
     "stylelint": "^14.3.0",
     "stylelint-config-prettier": "^9.0.3",
     "stylelint-config-recommended": "^6.0.0",
     "stylelint-config-standard": "~24.0.0",
     "stylelint-prettier": "^2.0.0",
-    "ts-jest": "^26.0.0",
-    "typescript": "~4.3.0-beta",
```

Comment:
Did removing typescript from the package.json help solve problems? Could it potentially hurt us later down the road not pinning the typescript version since we are getting so many typescript errors trying to run the readme commands?
I tried this for the jupyter-server-extension library and got `/bin/sh: tsc: command not found`

Reply:
It should install the version specified by jupyterlab-tour, which is: `"typescript": "~4.1.3"`. I don't think we want to pin more than the major and minor version numbers. But if you're running into issues building this, we might want to explicitly install that version of typescript.

Reply:
Got this working by adding "typescript": "~4.1.3" (if I tried without this I got `/bin/sh: tsc: command not found` for maap help), using the environment.yml that you gave me, and using the node version you used: v16.10.0
I will approve if adding that works for you too 
Also, what are your thoughts on adding a .nvmrc file with the node version 16.10.0?

Reply:
Note that I was able to replicate this a second time following the same method I described above with a new conda environment (but still created from your environment.yml)

Reply:
I've added the explicit install of typescript and everything looks good on my end. I will publish a new version of the extension after merging.